src/lumigo_opentelemetry/instrumentations/__init__.py

```diff
@@ -12,10 +13,24 @@ class AbstractInstrumentor(ABC):
     def __init__(self, instrumentation_id: str):
         self._instrumentation_id = instrumentation_id
 
+    def is_applicable(self) -> bool:
```

Comment:
if your'e changing `check_if_applicable` to `assert_instrumented_package_importable`, it might be worth to change this methods name as well.

Reply:
This method doesn't actually asserts (throws error), but returns a boolean - therefore I did not rename it. Or did I get you wrong?

Reply:
Looking at this again I think I misunderstood. Sorry about that.